RealmSwift-swift2.2/Object.swift

```diff
@@ -54,7 +54,8 @@ import Realm.Private
  optional numeric value.
 
  All property types except for `List` and `RealmOptional` *must* be declared as `dynamic var`. `List` and
- `RealmOptional` properties must be declared as non-dynamic `let` properties.
+ `RealmOptional` properties must be declared as non-dynamic `let` properties. Swift `lazy` properties are not allowed,
+ unless they are added to the ignored properties list.
```

Comment:
This part isn't specific to `lazy`. There should probably just be another paragraph that says that the restrictions don't apply to ignored properties.